.github/workflows/tests.yml

```diff
@@ -122,6 +124,19 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: ["3.11"]
+        sphinx-version: [""]
+        include:
+          # TODO: Cant build on oldest version (4.2) because of ablog,
+          # so we test on 5.0 in github_actions_install.sh
+          - os: ubuntu-latest
+            python-version: "3.8"
+            sphinx-version: "old"
+          # TODO: Can't build on dev because of dependency resolution :(
+          # - os: ubuntu-latest
+          #   python-version: "3.11"
+          #   sphinx-version: "dev"
```

Comment:
Also can't test the doc build job on sphinx dev because of some dependency problem. Something to look at in the future since catching Sphinx warnings early (as it says are checked in this job) is pretty important I think

Reply:
argh. yes, definitely want to test against sphinx dev; the events of this week make it obvious that we should have been doing that all along.